configure.ac

```diff
@@ -139,6 +139,19 @@ if test "$ENABLED_DISTRO" = "yes"
 then
     enable_shared=yes
     enable_static=yes
+    enable_all=yes
+fi
+AM_CONDITIONAL([BUILD_DISTRO], [test "x$ENABLED_DISTRO" = "xyes"])
+
+
+# ALL FEATURES
+AC_ARG_ENABLE([all],
+    [  --enable-all            Enable all wolfSSL features (default: disabled)],
```

Comment:
I'll make sure and update this one with more info. Yeah the entire configure.ac should have a cleanup to use AS_HELP_STRING.